.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -42,16 +44,21 @@ steps:
       -e SIG_GALLERY_NAME=${SIG_GALLERY_NAME} \
       -e SIG_IMAGE_NAME=${SIG_IMAGE_NAME} \
       -e SIG_IMAGE_VERSION=${SIG_IMAGE_VERSION} \
-      -e HYPERV_GENERATION="V1" \
+      -e HYPERV_GENERATION=${HYPERV_GENERATION} \
       -e VNET_RESOURCE_GROUP_NAME=${VNET_RESOURCE_GROUP_NAME} \
       -e WINDOWS_PATCH_URL=${WINDOWS_PATCH_URL} \
       -e WINDOWS_BASE_IMAGE_URL=${WINDOWS_BASE_IMAGE_URL} \
       -e WINDOWS_NANO_IMAGE_URL=${WINDOWS_NANO_IMAGE_URL} \
       -e WINDOWS_CORE_IMAGE_URL=${WINDOWS_CORE_IMAGE_URL} \
       ${CONTAINER_IMAGE} make -f packer.mk run-packer-windows
     displayName: Building windows VHD
+
+# Note: use -a to grep MANAGED_SIG_ID (packer-output should be read as a binary file in Linux)
   - script: |
+      echo MODE=$(MODE) && \
```

Comment:
remove line 58

Reply:
?

Reply:
You have changed the file several times so the comment is not correct now

Reply:
Yeah. I rebased a couple times. Did you remember what was the issue?

Reply:
I cannot remember it now. let us close this for now.